### ISSUE TITLE
Consider lightweight version of dataset for faster model training

### ISSUE THREAD

**Issue by florian-huber** [carpentries-incubator/deep-learning-intro#105]
When building episode 3 and the weather prediction dataset I felt the compuational cost was OK-ish (no training took longer than 1-2 minutes on my laptop). But @colinsauze and @dafnevk correctly pointed me to the fact that this might take much longer on other computers, more like 10 minutes which could already make it hard to run live.

One possible solution would be to either
1. Create a lightweight version of the dataset (less features), or
2. Make a subset of the dataset before training (@colinsauze suggested this, and it indeed seems like this could be a good learning experience, too).

**Comment by psteinb** [comment]
"premature optimisation is the root of many evil" ... what exactly took quite long?

**Comment by florian-huber** [comment]
It is the actual model training. To nicely demonstrate overfitting we have to train enough epochs and that can take a while (up to a few minutes per training). 

For now I tested removing about 50% of the features, which gives nearly identical results for the code in episode 3.
I didn't do a proper performance test, but since we work with a *very* shallow network I expect that this will speed up the model training by about 2x.

What I removed were locations far away from the center and everything wind related:
(sorry, just very quickly written code here)
```python
keep_columns = [x for x in data.columns if x.split("_")[0] not in ["BUDAPEST", "LJUBLJANA", "MONTELIMAR", "OSLO", "PERPIGNAN", "ROMA", "STOCKHOLM"]]
keep_columns= [x for x in keep_columns if "wind" not in x]
drop_columns = [x for x in data.columns if x not in keep_columns]

X_data = data.loc[:365*3].drop(columns=['DATE', 'MONTH']+drop_columns)
```
